src/main/java/com/util/ServletUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.util;
+
+import com.entities.Organization;
+import com.google.gson.Gson;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public final class ServletUtil {
+
+    public static <T> T expect(HttpServletRequest request , Class<T> Class) throws IOException {
+
+        Gson gson = new Gson();
+        T t = gson.fromJson(request.getReader(), Class);
+        return t;
+
+    }
+
+    public static void response (Organization organization ,HttpServletResponse response) throws IOException {
```

Comment:
This function should allow any type of object. 
```suggestion
    public static void response (HttpServletResponse response, Object payload) throws IOException {
```

Reply:
Also let's change the name to `respond` (function names should always be a verb)